### PR TITLE
remove dependency on bash, use python to do the downloading

### PR DESCRIPTION
**Pull request by andrewkittredge**
## Description of proposed changes
Does not work on Windows as is, because it calls out to bash
## Related issue(s)

Fixes # (issue)

## Checklist

Need help on these? Just ask!

* [x] I have read the **CONTRIBUTING** document.
* [ ] I have verified that my changes are covered by continuous integration.
* [ ] All new and existing tests passed.

### CODE REVIEW

File: spouse/utils.py [hunk 1]
```diff
@@ -46,3 +46,25 @@ def load_data() -> Tuple[
...
+        if not os.path.exists(filename):
+            reload = True
+    if reload:
+        if os.path.exists("data/"):
+            shutil.rmtree("data/")
+            os.mkdir("data")
+        urllib.request.urlretrieve(DATA_URL, "data.zip")       
+        with zipfile.ZipFile("data.zip", "r") as zip_ref:
+            zip_ref.extractall("data")
+        os.remove("data.zip")
```
Comment (vincentschen) [inline spouse/utils.py]: can we refactor this to a separate file that can be run from the CLI? 
that way, folks don't need to be _in_ the notebook in order to download the data

Comment (vincentschen) [reply spouse/utils.py]: also, any chance this script can be shared across different tutorials?

**Comment by vincentschen** [comment]
Hi @andrewkittredge — we've [updated the tutorials](https://github.com/snorkel-team/snorkel-tutorials/pull/173) so that tests on `master` should be passing. You should be good to go on rebasing this PR!

**Comment by andrewkittredge** [comment]
I will look.

**Comment by andrewkittredge** [comment]
I got rid of all the bash, I have only tested this on Windows.  To import the download function from all the directories I had to modify the path, is there a better way to do that?

File: spouse/tf_model.py [hunk 2]
```diff
@@ -65,5 +65,5 @@ def get_model(
     layer = Dense(32, activation=tf.nn.relu)(layer)
     probabilities = Dense(2, activation=tf.nn.softmax)(layer)
     model = tf.keras.Model(inputs=[bet_ph, left_ph, right_ph], outputs=probabilities)
-    model.compile(tf.train.AdagradOptimizer(0.1), "categorical_crossentropy")
+    model.compile(tf.optimizers.Adam(0.1), "categorical_crossentropy")
```
Comment (vincentschen) [inline spouse/tf_model.py]: to keep this PR in scope, can we remove this change? or put it in a different PR?
